clients/web/src/stylesheets/widgets/accessWidget.styl

```diff
@@ -28,6 +28,21 @@
   font-weight bold
   padding 3px 10px
 
+i.g-flag-icon
+  position relative
+
+.g-flag-count-indicator
+  display inline-block
+  border-radius 3px
+  background-color rgba(200, 0, 0, 0.75)
```

Comment:
I'm not sure that there are enough pixels at these sizes to easily see the flag behind a transparent badge, so yes, I'd suggest removing the alpha.